new_dawn_server/questions/api/resources.py

```diff
@@ -0,0 +1,40 @@
+from new_dawn_server.questions.models import AnswerQuestion, Question
+from new_dawn_server.users.api.resources import UserResource
+from tastypie import fields
+from tastypie.authorization import Authorization
+from tastypie.resources import ModelResource, ALL_WITH_RELATIONS
+
+ANSWER_QUESTION_FIELDS = {
+    "answer": True,
+    "order": True,
+}
+
+QUESTION_FIELDS = {
+    "question": True,
+    "sample_answer": True,
+}
+
+
+class QuestionResource(ModelResource):
+    class Meta:
+        always_return_data = True
+        authorization = Authorization()
+        allowed_methods = ["get", "post"]
```

Comment:
Not sure if we want user to be able to create questions. But we can keep this for now.